app/models/fruit_ticket_transaction.rb

```diff
@@ -0,0 +1,22 @@
+class FruitTicketTransaction < ApplicationRecord
+  belongs_to :from_user # null means the website gave you fruit tickets
+  belongs_to :to_user # null means bought something from the website
+
+  # how do you get fruit tickets
+  #   - people listen to your show
+  #   - 1 fruit ticket for every listener?
```

Comment:
this seems like a hard thing to calculate because of the existing issue(s) of getting a listener count...
maybe instead of listeners, it's users in the chat? or maybe even just registered users?

Reply:
Yeah users in the chat seems like a better metric.